### PR TITLE
build: Update wrangler

### PR DESCRIPTION
**Pull request by istarkov**
## Description

1. What is this PR about (link the issue and add a short description)

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -9704,11 +9737,6 @@ packages:
     resolution: {integrity: sha512-TkRKr9sUTxEH8MdfuCSP7VizJyzRNMjj2J2do2Jr3Kym598JVdEksuzPQCnlFPW4ky9Q+iA+ma9BGm06XQBy8g==}
     engines: {node: '>= 0.6'}
 
-  typescript@5.4.5:
```
Comment (TrySound) [inline pnpm-lock.yaml:9707]: good

File: packages/cli/templates/cloudflare/package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
   },
   "devDependencies": {
     "@cloudflare/workers-types": "^4.20240405.0",
-    "wrangler": "^3.48.0",
+    "wrangler": "^3.63.2",
     "miniflare": "^3.20231030.4"
```
Comment (TrySound) [inline packages/cli/templates/cloudflare/package.json:21]: maybe bump mini flare and worker-types as well?

Comment (istarkov) [reply packages/cli/templates/cloudflare/package.json]: removed miniflare, bumped types

File: fixtures/webstudio-cloudflare-template/package.json [hunk 3]
```diff
@@ -47,13 +47,13 @@
     "zod": "^3.22.4"
   },
   "devDependencies": {
-    "@cloudflare/workers-types": "^4.20240405.0",
+    "@cloudflare/workers-types": "^4.20240620.0",
     "@remix-run/dev": "2.10.0",
     "@types/react": "^18.2.70",
     "@types/react-dom": "^18.2.25",
     "miniflare": "^3.20231030.4",
```
Comment (TrySound) [inline fixtures/webstudio-cloudflare-template/package.json:54]: and here pls

**Review by TrySound — APPROVED** [review]

**Pull request merged**